README.md

```diff
@@ -59,27 +59,34 @@ class { 'seed_stack::worker':
 ```
 **NOTE:** For combination controller/workers it is necessary to set `controller_worker => true` for both the `seed_stack::controller` class and the `seed_stack::worker` class so that the two classes do not conflict.
 
-#### Load balancers
-If you need to expose apps running on Seed Stack to the outside world, Nginx can be used to load-balance between app instances and expose the apps from an accessible endpoint. To set up Nginx to be dynamically configured to do this, use the `seed_stack::load_balancer` class.
+#### External load balancers and routers
+We use Nginx to load balance and route between containers. Sometimes it is useful to do this outside of the Mesos cluster itself. For instance, an external host could be the load balancer for a web service or some service running on the host could need to be routed to containers directly.
 
+Nginx is dynamically-configured using Consul Template in these cases. First, set that up:
+```puppet
+class { 'seed_stack::template_nginx':
+  consul_address => '192.168.0.5', # Consul address for Consul Template to connect to
+}
+```
+Then you can set up either set up a load balancer or a router (or both):
 ```puppet
 include seed_stack::load_balancer
+include seed_stack::router
 ```
-Different parts of the class can be disabled if the node it is being included on already has Nginx or Consul Template installed. See the [manifest source](manifests/load_balancer.pp) for more information.
 
 #### Consul DNS
 It's often useful to use Consul's DNS for service discovery on nodes that aren't controllers or workers. For example, a database node could advertise it's service to other nodes using Consul. To do this, use the `seed_stack::consul_dns` class. The class needs a few parameters so that it knows how to join the Consul cluster:
 ```puppet
 class { 'seed_stack::consul_dns':
   advertise_addr => $ipaddress_eth0, # Address to advertise for services on this node
-  join => ['10.215.32.11', '10.215.32.12'], # List of any Consul nodes already in the cluster
+  join           => ['10.215.32.11', '10.215.32.12'], # List of any Consul nodes already in the cluster
 }
 
 consul::service { 'postgresql':
-  port => 5432,
+  port   => 5432,
   checks => [
     {
-      script => '/usr/bin/pg_isready',
+      script   => '/usr/bin/pg_isready',
       interval => '30s',
     }
   ]
```

Comment:
Maybe condense this a bit by putting the braces and brackets on the same line?
```
  checks => [{
    ...
  }],
```

Also, a trailing comma is probably a good idea for consistency.